cs_practical_examples/4w-marketing/4w_after_2.js

```diff
@@ -0,0 +1,13 @@
+<div class=”4wNET hidden” style=”display:none”>
+    <script type="text/javascript">
+       	m3ads_system = "4WM";
+       	m3ads_partnernumber = 5000;
+   	m3ads_containerclass = "container class";
+m3ads_numberadverts = x;
+m3ads_logoimagewidth = xx;
+m3ads_logoimageheight = xx;
+m3ads_subpartner = "ads_subpartner";
+m3ads_customheader = x;
+    </script>
+    <script  type="text/javascript" src="//cdn.feed.4wnet.com/resources/scripts/jsAds-1.4.min.js"></script>
```

Comment:
@giniyat202 shouldn't this be blocked (I don't know the context of the example)?

Reply:
Looks like the original example hides the banner rather than blocking it
![image](https://user-images.githubusercontent.com/3234422/92735196-810e1e80-f379-11ea-95fc-83e4178e9eb4.png)
I just left it as it is